scorecard/Makefile

```diff
@@ -5,8 +5,9 @@ PROJECT = $(shell gcloud config get-value project)
 SCORECARD_IMAGE = gcr.io/$(PROJECT)/scorecard:$(shell docker images -q --no-trunc scorecard:latest | sed -e 's,[^:]*:,,')
 
 build:
+	make -C ../docker build
 	-docker pull gcr.io/$(PROJECT)/scorecard
-	docker build . -t scorecard --cache-from scorecard,base,ubuntu:18.04
+	docker build -t scorecard --cache-from scorecard,base,ubuntu:18.04 .
```

Comment:
This should probably include `gcr.io/$(PROJECT)/scorecard:latest` in preference to `scorecard` (which almost certainly is unavailable on the build machine). `deploy` should push latest.